packages/replayer/src/assets/snapshot-assets.ts

```diff
@@ -1,5 +1,5 @@
 import { mkdir, writeFile } from "fs/promises";
-import { dirname, join } from "path";
+import path, { dirname, extname, join } from "path";
```

Comment:
Apparently imports should be optimized here.

Reply:
Ah yes! Not sure how that slipped in. I'll fix.

Reply:
```suggestion
import { dirname, extname, join } from "path";
```